packages/data/addon/mirage/handlers/graphql.js

```diff
@@ -11,6 +11,13 @@ import moment from 'moment';
 const API_DATE_FORMAT = 'YYYY-MM-DD';
 const ASYNC_RESPONSE_DELAY = 5000; // ms before async api response result is populated
 
+function _getSeedForRequest(table, args, fields) {
```

Comment:
Could move this function up to data/mirage, but this should be fine too

https://github.com/yahoo/navi/blob/dd761f5cd895efb5aa63bd8efe1fd868428477e4/packages/app/mirage/config.js#L17-L29

Reply:
I originally looked into doing something like this, but it's a bit more than what's needed for this instance. We just need a deterministic way of seeding faker based on the request. If it needed to be a unique seed for each request, then I would've used a proper hashing method.